src/js/components/homepageUpdate/HomepageResources/HomepageResources.jsx

```diff
@@ -60,11 +60,11 @@ const cardObjects = [
                 <FontAwesomeIcon icon="sitemap" color="#0081a1" size="lg" />
             </div>
         ),
-        headline: 'Data Model',
-        text: 'Learn how our data is organized',
+        headline: 'About the Data',
+        text: 'Read important data disclosures',
         buttonText: (
             <>
-                View the model&nbsp;&nbsp;
+                View the disclosures;
```

Comment:
the two &nbsp need to be here for spacing between the word and the arrow

Reply:
fixed